src/state/slices/swappersSlice/selectors.ts

```diff
@@ -86,3 +86,5 @@ export const selectBuyAssetUsdRate = createSelector(
     return buyAssetMarketData.price
   },
 )
+
+export const selectIsDonating = createSelector(selectSwappers, swappers => swappers.isDonating)
```

Comment:
Maybe `selectWillDonate`, or something. "donating" is very present participle verby.